src/bundle/Resources/views/themes/admin/user/role_assignment/create.html.twig

```diff
@@ -34,9 +34,9 @@
                                 id: user.content.versionInfo.contentInfo.id,
                                 name: ibexa_content_name(user)
                             }) %}
-                            {% set items_map = form.users.vars.data|reduce((output, user) => output|merge({
-                                "{{user.content.versionInfo.contentInfo.id}}": user.id
-                            }), {}) %}
+                            {% set items_map = form.users.vars.data|reduce((output, user) => output + {
+                                (user.content.versionInfo.contentInfo.id): user.id,
```

Comment:
Note: `“{{ some_variable }}“` solution does not work, variable should be surrounded with parentheses.